Makefile

```diff
@@ -7,12 +7,19 @@ DOCKER_IMAGE_SHELLCHECK ?= koalaman/shellcheck:v0.8.0@sha256:4c4427336d2b4bdb054
 # Contains chart-testing cli,  helm and kubectl.
 DOCKER_IMAGE_CT ?= quay.io/helmpack/chart-testing:v3.7.0@sha256:2f87e56a0cebc6d9bb78d51cb6adac97858afb0887cac4d65f7f8c6a16054568
 
-VERIFY_SCRIPTS += /src/tests/e2e-kind.sh
+# Official image of chart-releaser[https://github.com/helm/chart-releaser].
+# Contains chart-releaser cli.
+DOCKER_IMAGE_CR ?= quay.io/helmpack/chart-releaser:v1.4.0@sha256:f9e6b06a1d60bcd94a2a6a4988a59233936239347c69b4ab2d99916ffc848a5b
+
+VERIFY_SCRIPTS += /src/tests/e2e-kind.sh /src/scripts/release.sh
```

Comment:
just a comment to what is it about

Reply:
it contains list of scripts to be verified. `lint-scripts` target will use that variable to execute shellcheck. Maybe we should change the naming.

Reply:
Comment is added.